generator-scrivito/generators/obj/index.js

```diff
@@ -0,0 +1,98 @@
+const Generator = require("yeoman-generator");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.log("Welcome in scrivito Obj/Widgets generator!");
```

Comment:
Since this generator is either called directly `yo scrivito:obj` or thru selecting the command (`yo` => "Scrivtio" => "Generate a new Obj")  I think we don't need to greet the person again. I suggest to just remove the `constructor` method completely.